test/tsconfig.json

```diff
@@ -7,7 +7,8 @@
     "baseUrl": "../",
     "sourceMap": true,
     "paths": {
-      "@salesforce/kit": ["node_modules/@salesforce/kit"]
+      "@salesforce/kit": ["node_modules/@salesforce/kit"],
+      "@salesforce/core": ["node_modules/@salesforce/core"]
```

Comment:
is this still needed outside of yarn link?

Reply:
Not sure, I'll check.